sklearn/metrics/cluster/_supervised.py

```diff
@@ -740,10 +740,12 @@ def mutual_info_score(labels_true, labels_pred, *, contingency=None):
     Parameters
     ----------
     labels_true : int array, shape = [n_samples]
-        A clustering of the data into disjoint subsets.
+        A clustering of the data into disjoint subsets, called $U$ in the
+        above formula.
```

Comment:
To render the math formatting.
```suggestion
        A clustering of the data into disjoint subsets, called :math:`U` in the
        above formula.
```

Reply:
Also maybe of the sentence in a paragraph above can be changed to use:
```
This metric is furthermore symmetric: switching :math:`U` (i.e 
``label_true``) with :math:`V` (i.e. ``label_pred``) will return the 
same score value.
```

Reply:
Also note that those patches can also be applied in some of the others metrics' signatures (e.g. the one of `normalized_mutual_info_score` for the math notation).